### ISSUE TITLE
always get loss not a number, is there something need to do when trainning?

### ISSUE THREAD

**Issue by botaichang** [zuokai/roialign#4]
when i trainning faster rcnn using roialign, i always get bbox_loss = nan, and cls_loss is very large, is there need other settings?  but when i use same configurations and replace roialign with roipooling, the loss converge.

**Comment by zuokai** [comment]
can i see your prototxt?

**Comment by zuokai** [comment]
sometimes i found the loss diveraged，you can set the multiple intepolate times by setting is_multiple_interpolate = true, if this step still failed, you can train with roipooling first and finetune the model with roialign

**Comment by botaichang** [comment]
I just replace ROIppoling layer with ROIAlign in the prototxt. Yeah, I train with roipooling first and then finetune with roialign, but MAP is very low.

**Comment by zuokai** [comment]
It‘s my experience with roialign：
1）MAP 4 times cubic > 1 time cubic ≈ 4 times bilinear > 1 time bilinear
2)  roialign is more effective with small objects and seems not very effective in large objects
I didn’t see the situation that MAP is very low，so I don‘t know what’s happened with your models
